react-app/src/features/package/package-details/hooks.tsx

```diff
@@ -122,11 +122,6 @@ export const recordDetails = (data: opensearch.main.Document): DetailSectionItem
       return !(data.actionType === "Extend");
     },
   },
-  {
-    label: "Status Date",
-    value: data.statusDate ? formatSeatoolDate(data.statusDate) : BLANK_VALUE,
-    canView: (u) => (!u || !u.user ? false : isCmsUser(u.user)),
-  },
```

Comment:
Should we ask whether super users or other types of users can view this information?

Reply:
@asharonbaltazar I asked and it should be removed altogether.